mucommander-protocol-nfs/src/main/java/com/sun/nfs/NfsSecurity.java

```diff
@@ -37,7 +37,6 @@
 
 package com.sun.nfs;
 
-import java.io.*;
 import java.util.*;
 import com.sun.rpc.*;
```

Comment:
what about removing wildcard imports completely?

Reply:
I would prefer to keep this code as close as possible to the original code that we cloned, I'll revert changes in "cloned" code

Reply:
OK. What about other places with wildcards?
FYI, I've just picked this file randomly to make a comment. 
BTW, such subsystems/packages could be marked as being cloned from elsewhere? For example in `package-info.java` with a link to a source?

Reply:
> OK. What about other places with wildcards? FYI, I've just picked this file randomly to make a comment.

ah ok. in general, I'm good with replacing imports with wildcards - the formatter also does it, but as we discussed once - I prefer not to have PRs with so many formatting changes as they are hard to review. PRs that just replace those imports or replacing them gradually would be ok

> BTW, maybe such subsystems/packages could be marked as being cloned from elsewhere? For example in `package-info.java` with a link to a source?

yeah, that sounds like a good idea

Reply:
> > BTW, maybe such subsystems/packages could be marked as being cloned from elsewhere? For example in `package-info.java` with a link to a source?
> 
> yeah, that sounds like a good idea

(for a separate PR)